src/styles.css

```diff
@@ -0,0 +1,158 @@
+body {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #6d6970;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  background-color: black;
+  position: fixed;
+  width: 100%;
+}
```

Comment:
- [ ] Home page header and navigation should be similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md). Kindly align the navbar elements to the left.

Your layout  | Wireframe layout
------------- | -------------
<img width="1407" alt="Screenshot 2022-05-20 at 18 05 09" src="https://user-images.githubusercontent.com/77038610/169545068-33d648aa-46ad-42b3-ab2e-b62cc5916594.png"> | <img width="535" alt="Screenshot 2022-05-20 at 18 04 48" src="https://user-images.githubusercontent.com/77038610/169545007-0c7a4d5c-88cd-4e89-9849-b36e7f3bbb1c.png">

I highly admire your creativity, and I personally think that your way of layout looks better, but for this project we should follow the requirements ..